salesforce_api/sfdc-new-contact-to-twilio-sms/sample.bal

```diff
@@ -0,0 +1,73 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/trigger.salesforce as sfdcListener;
+import ballerinax/twilio;
+
+type SalesforceListenerConfig record {
+    string username;
+    string password;
+};
+
+type TwilioClientConfig record {
+    string accountSId;
+    string authToken;
+};
+
+const string COMMA = ",";
+const string EQUAL_SIGN = "=";
+const string CLOSING_BRACKET = "}";
+const string NO_STRING = "";
+const string CHANNEL_NAME = "/data/ContactChangeEvent";
+
+// Salesforce configuration parameters
+configurable SalesforceListenerConfig salesforceListenerConfig = ?;
+
+// Twilio configuration parameters
+configurable TwilioClientConfig twilioClientConfig = ?;
+configurable string fromNumber = ?;
+configurable string toNumber = ?;
+
+listener sfdcListener:Listener sfdcEventListener = new ({
+    username: salesforceListenerConfig.username,
+    password: salesforceListenerConfig.password,
+    channelName: CHANNEL_NAME
+});
+
+service sfdcListener:RecordService on sfdcEventListener {
+    remote function onCreate(sfdcListener:EventData payload) returns error? {
+        string firstName = NO_STRING;
+        string lastName = NO_STRING;
+        map<json> contactMap = payload.changedData;
+        string[] nameParts = re `,`.split(contactMap["Name"].toString());
```

Comment:
```suggestion
        string[] nameParts = re `,`.split(payload.changedData["Name"].toString());
```

Reply:
Updated in [bcf09ac](https://github.com/ballerina-guides/integration-samples/pull/31/commits/bcf09ac42e87543485854586312facacacd050cb)